src/components/Image.ts

```diff
@@ -46,21 +54,39 @@ export class Image extends Component<ImageProps, ImageChild> {
 	public static readonly mixed: boolean = false;
 
 	#relationshipId: string | null = null;
+	#relationshipIdSvg: string | null = null;
```

Comment:
I can appreciate that you introduced `dataExtensions`, it should be relatively easy to add additional extensions to it as well. However, the data belonging to each extension is then ploinked onto `Image` as `relationshipIdSvg`, `locationSvg`, etc. It appears that the `dataExtensions` prop and the class properties do not "scale" in the same way.

My suggestion would be to introduce a `dataExtensions` property on the `Image` class (read: not on `Image.props`), and place all things that need to be known about an extension there. For example,

```
#dataExtensions: {
  svg?: {
    data: string;
    location: string;
    mime: string;
  }
}
```